lua/tests/dotenv/parser.lua

```diff
@@ -90,16 +93,6 @@ return {
 					.to.beA("table")
 			end
 		},
-		{
```

Comment:
Keep this test, it provides a clearer indication on what requirement we're validating.